docs/source/cli.rst

```diff
@@ -29,13 +29,17 @@ and list experiments, and download artifacts.
       ui           Run the MLflow tracking UI.
 
 
-Each individual command has a detailed help screen accessible via ``mlflow command_name --help``
+Each individual command has a detailed help screen accessible via ``mlflow command_name --help``.
 
+.. contents:: Table of Contents
+  :local:
+  :depth: 1
 
-Azure ML
---------
 
-Subcommands to serve models on Azure ML.
+Azure Machine Learning Models
```

Comment:
Not in love with how this looks, but it's not a regression:
![image](https://user-images.githubusercontent.com/1400247/45247949-7fa84800-b2c0-11e8-8c1b-d198fa8bce7e.png)